templates/seL4VMParameters.template.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <camkes.h>
+#include <arm_vm/gen_config.h>
+
+/*- if configuration[me.name].get('vm_address_config') -*/
+
+unsigned long ram_base = /*? configuration[me.name].get('vm_address_config').get('ram_base') ?*/;
+unsigned long ram_paddr_base = /*? configuration[me.name].get('vm_address_config').get('ram_paddr_base') ?*/;
+unsigned long ram_size = /*? configuration[me.name].get('vm_address_config').get('ram_size') ?*/;
+unsigned long ram_offset = /*? configuration[me.name].get('vm_address_config').get('ram_offset') ?*/;
+unsigned long dtb_addr = /*? configuration[me.name].get('vm_address_config').get('dtb_addr') ?*/;
+unsigned long initrd_max_size = /*? configuration[me.name].get('vm_address_config').get('initrd_max_size') ?*/;
+unsigned long initrd_addr = /*? configuration[me.name].get('vm_address_config').get('initrd_addr') ?*/;
+
+/*- if configuration[me.name].get('vm_address_config').get('kernel_entry_addr') == -1 -*/
```

Comment:
```suggestion
/*- if configuration[me.name].get('vm_address_config').get('kernel_entry_addr') != -1 -*/
```

Reply:
Nice catch. I also had to check for `'-1'`  since its technically a string